### PR TITLE
Usecase docs: AWS is a supported KMS

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

**Comment by k-wall** [comment]
I think once this is review, we merge once the release is made.

File: _use_cases/encryption-at-rest.markdown [hunk 1]
```diff
@@ -84,7 +84,7 @@ The solution's foundations rest on industry-standard encryption techniques.
 
 * Envelope Encryption is employed to efficiently encrypt/decrypt records. Envelope Encryption is specified by [NIST SP.-800-57 part 1 revision 5](https://csrc.nist.gov/Projects/Key-Management/Key-Management-Guidelines).
 * Integrates with common Key Management Services for safe and secure storage of key encryption keys.
-  * Plugin for [HashiCorp Vault](https://www.hashicorp.com/)&#174; 
+  * Plugin for [HashiCorp Vault](https://www.hashicorp.com/)&#174; and [AWS Key Management Service](https://docs.aws.amazon.com/kms/latest/developerguide/overview.html)&#174;.
```
Comment (SamBarker) [inline _use_cases/encryption-at-rest.markdown:87]: nit:
```suggestion
  * Plugins for [HashiCorp Vault](https://www.hashicorp.com/)&#174; and [AWS Key Management Service](https://docs.aws.amazon.com/kms/latest/developerguide/overview.html)&#174;.
```

File: assets/pages/images/encryption-at-rest_solution.png
Comment (SamBarker) [inline assets/pages/images/encryption-at-rest_solution.png:1]: The width and height of the PNG double in this PR (I'm not particularly worried by the file size) but does that mean we need to change the dimensions at which we load the image into the use cases page?

**Review by SamBarker — APPROVED** [review]
I haven't checked the rendered view but change looks good to me.

File: assets/pages/images/encryption-at-rest_solution.png
Comment (k-wall) [reply assets/pages/images/encryption-at-rest_solution.png:1]: I'll check.

**Pull request merged**